src/components/IconPicker.tsx

```diff
@@ -52,15 +52,21 @@ const IconPicker = ({ schemaType, value = {}, onChange }: ObjectInputProps) => {
     setSelected(null);
   };
 
-  const setIcon: SearchResultsOnSelectCallback = (icon) => {
+  const setIcon: SearchResultsOnSelectCallback = (
+    icon: IconObject,
+    ele: HTMLButtonElement
+  ) => {
     if (selected && icon.name === selected.name) return unsetIcon();
 
+    const svgString = ele.getElementsByTagName('svg')[0].outerHTML;
```

Comment:
Can we move this into its own function so that it's not run if storeSvg option is not being used? This is not a big thing really but my thinking is that we want to abstract this new feature to when svgStore is true in case this results in any errors which we have not yet discovered, which would in that case break everything related to actually storing the icon.

```suggestion
    const svgString = () => ele.getElementsByTagName('svg')[0].outerHTML;
```